filter/ArticleCrossrefXmlFilter.php

```diff
@@ -268,19 +241,28 @@ public function createJournalArticleNode($doc, $submission)
             }
         }
 
+        // ai:program
```

Comment:
What is `ai:program` are these referring to codes from the Crossref schema? If so, a note above or at the first one would be helpful.

Reply:
Yes, the crossref element is called so -- maybe I can add some more explanation here